### PR TITLE
Nl add nl splash pages

### PR DESCRIPTION
**Pull request by JoeriVankeirsbilck**
Hi team,

My first (debugged hopefully) pull request for the NL splash pages - all feedback welcomed so I can improve on this.
@octaveauger @greysteil @Ken-Lo  Who else should be mentioned in this request?

Thanks,
Joeri

### CODE REVIEW

**Comment by greysteil** [comment]
@jamesshedden - can you take a look?

File: app/constants/team.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const team = [
   'Corinne Thompson',
   'Matt Thomson',
   'Côme Trémeau',
+  'Joeri Vankeirsbilck',
```
Comment (jamesshedden) [inline app/constants/team.js]: @JoeriVankeirsbilck :tada:

Comment (JoeriVankeirsbilck) [inline app/constants/team.js]: @jamesshedden Do I need to action, i.e. remove my name, for the update to process correctly?

Comment (jamesshedden) [inline app/constants/team.js]: @JoeriVankeirsbilck No no, just celebrating the addition of your name to the team page! Everything good :+1:

**Comment by jamesshedden** [comment]
@greysteil Sure thing!

**Comment by jamesshedden** [comment]
@JoeriVankeirsbilck Hi Joeri, awesome work sorting this all out! Code is mostly fine - just a few things that it'd be good to catch up about, I'll ping you on Slack to arrange a quick chat rather than commenting further here :+1:

File: app/messages/nl-NL.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    per_transaction_amount_normal: '1%',
+    pro_monthly_fee: '€250',
+    normal_capped_at_amount: '€200',
+    pro_number_of_monthly_payments: '> 500',
+  },
+  signin: {
+    path: 'https://manage.gocardless.com',
+  },
+  signup: {
+    path: 'https://manage.gocardless.com/signup',
```
Comment (Ken-Lo) [inline app/messages/nl-NL.js]: This needs to be https://manage.gocardless.com/signup?lang=nl following the merge of https://github.com/gocardless/splash-pages/pull/305 :-)

**Comment by jamesshedden** [comment]
@Ken-Lo I didn't see your comments when going through stuff with Joeri so if you're cool to push up any changes you need to make separately (or go through with Joeri) that'd be awesome.

Other than that, :+1: from me!

**Comment by Ken-Lo** [comment]
Nice! I've just made a few updates to the message files - happy to merge this up as is.

@JoeriVankeirsbilck Anything else you want to do before we merge this in? Homepage USPs still need to be reviewed but I think you're planning to do this later on at another time?

**Comment by JoeriVankeirsbilck** [comment]
Fantastic work team, thanks - I'm good to have it go live!

USPs will be changed as we get the positioning and messaging right for
Netherlands in the next few days and weeks.

On 22 October 2015 at 14:22, Ken-Lo <notifications@github.com> wrote:

> Nice! I've just made a few updates to the message files - happy to merge
> this up as is.
>
> @JoeriVankeirsbilck <https://github.com/JoeriVankeirsbilck> Anything else
> you want to do before we merge this in? Homepage USPs still need to be
> reviewed but I think you're planning to do this later on at another time?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gocardless/splash-pages/pull/320#issuecomment-150222068>
> .
>



-- 
*Joeri Vankeirsbilck*
Country Lead The Netherlands & Belgium
GoCardless.com <http://gocardless.com/> | m +44 77 2976 7567 | t +44 20
3744 3259

**Comment by JoeriVankeirsbilck** [comment]
Thanks team, great work!

On 22 October 2015 at 15:49, Ken-Lo <notifications@github.com> wrote:

> Merged #320 <https://github.com/gocardless/splash-pages/pull/320>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gocardless/splash-pages/pull/320#event-442733053>.
>



-- 
*Joeri Vankeirsbilck*
Country Lead The Netherlands & Belgium
GoCardless.com <http://gocardless.com/> | m +44 77 2976 7567 | t +44 20
3744 3259

**Pull request merged**